build.gradle.kts

```
@@ -22,6 +22,7 @@ dependencies {
     implementation("org.jetbrains.kotlin:kotlin-reflect")
     implementation("org.jetbrains.kotlin:kotlin-stdlib-jdk8")
 
+    implementation("org.springframework.boot:spring-boot-starter-validation:2.6.3")
     implementation("org.springframework.cloud:spring-cloud-starter-aws-messaging:2.2.6.RELEASE")
```

```suggestion
    implementation("org.springframework.cloud:spring-cloud-starter-aws-messaging:${Libs.AWS_MESSAGING}")
```
<br/>
다음과 같이 `buildSrc`를 활용하거나 아니면 아래와 같이 `gradle.properties`를 이용해서 버전관리를 하는 것이 좋을 것 같아요. 

gradle.properties
```
awsMessagingVersion = 2.2.6
```

build.gradle.kts
```kotlin
val awsMessagingVersion by project
```

```suggestion
    implementation("org.springframework.cloud:spring-cloud-starter-aws-messaging:${awsMessagingVersion}")
```

---

저렇게 관리하면 다른 AWS 라이브러리들을 사용할때도 동일한 변수를 사용하면되니까 버전 관리가 편리해지는건가?

---

kotlin dsl 관련해서 읽을만한 레퍼런스좀 주세용 ㅜㅜ

---

https://docs.gradle.org/current/userguide/organizing_gradle_projects.html#sec:build_sources
요기용